queue_services/events-listener/src/events_listener/worker.py

```diff
@@ -32,6 +32,7 @@
 import nats
 from auth_api.models import Org as OrgModel
 from auth_api.models import db
+from auth_api.services import Flags
```

Comment:
@seeker25 in the future we might wanna put this Flags service in the shared common BE repo or even the entity_queue_common so that other things can use it like this without coupling with one of the apis